Kitodo/src/main/java/org/kitodo/production/forms/ProzesskopieForm.java

```diff
@@ -498,7 +511,31 @@ protected void removeCollectionsForChildren(LegacyMetsModsDigitalDocumentHelper
      * Create the process and save the metadata.
      */
     public String createNewProcess() {
+        if (Objects.nonNull(titleRecordLinkTab.getTitleRecordProcess())) {
+            if (Objects.isNull(titleRecordLinkTab.getInsertionPositionSelectionSelectedItem())) {
+                Helper.setErrorMessage("prozesskopieForm.createNewProcess.noInsertionPositionSelected");
+                return stayOnCurrentPage;
+            } else {
+                User titleRecordOpenUser = DataEditorForm
+                        .getUserOpened(titleRecordLinkTab.getTitleRecordProcess().getId());
+                if (Objects.nonNull(titleRecordOpenUser)) {
+                    Helper.setErrorMessage("prozesskopieForm.createNewProcess.titleRecordOpen",
+                        titleRecordOpenUser.getFullName());
+                    return stayOnCurrentPage;
+                }
+            }
+        }
         if (createProcess()) {
+            if (Objects.nonNull(titleRecordLinkTab.getTitleRecordProcess())) {
+                ServiceManager.getProcessService().refresh(prozessKopie);
```

Comment:
The reload is performed because the ID of the process is accessed three lines lower. The ID is first `null` when the object is created and is assigned by Hibernate when the process is saved, but not automatically set in the object in memory. By reloading the object, I make sure that the ID in the object is available in memory.
The function `createProcess()` two rows higher (in the `if` condition) creates the object in the database, so it has already been created at this point.